dbadapter/dbadapter.go

```diff
@@ -75,12 +80,9 @@ func ConnectToDBClient(dbName string, url string, enableStream bool, nfProfileEx
 
 	if nfProfileExpiryEnable {
 		logger.AppLog.Infoln("NfProfile document expiry enabled")
-		ret := db.RestfulAPICreateTTLIndex("NfProfile", 0, "expireAt")
-		if ret {
-			logger.AppLog.Infoln("ttl Index created for Field : expireAt in Collection: NfProfile")
-		} else {
-			logger.AppLog.Infoln("ttl Index exists for Field : expireAt in Collection: NfProfile")
-		}
+		ttlIndexCreated := db.RestfulAPICreateTTLIndex("NfProfile", 0, "expireAt")
+		ttlIndexStatus := ttlIndexStatusMap[ttlIndexCreated]
```

Comment:
I am not sure about creating a map for this, we could use a simple if here:

```suggestion
		ttlIndexStatus := "exists"
		if ttlIndexCreated {
		    ttlIndexStatus = "created"
		}
```

It would get rid of the map and be more localized, wdyt?

Reply:
Agree. It was the first approach I took but later I decided to completely get rid of the `if` statement. So, I used the map instead of the `if`. I just "reverted" the PR to with your suggestion